### PR TITLE
Added package.xml <2.0.x> [9836]

### PR DESCRIPTION
**Pull request by MiguelCompany**
This PR adds a `package.xml` file to ease the integration with ROS 2.

### CODE REVIEW

**Comment by MiguelCompany** [comment]
It would be great if someone @ros2 reviews this

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fastrtps</name>
+  <version>2.1.0</version>
```
Comment (IkerLuengo) [inline package.xml:5]: `2.0.1`

**Review by IkerLuengo — CHANGES_REQUESTED** [review]

File: package.xml (hunk 1 above)
Comment (MiguelCompany) [reply package.xml:5]: 🤦

**Review by IkerLuengo — APPROVED** [review]
LGTM

**Pull request merged**